mitiq/benchmarks/randomized_benchmarking.py

```diff
@@ -47,35 +55,56 @@ def generate_rb_circuits(
             "Only generates RB circuits on one or two "
             f"qubits not {n_qubits}."
         )
-    qubits = LineQubit.range(n_qubits)
+    qubits = cirq.LineQubit.range(n_qubits)
     cliffords = _single_qubit_cliffords()
-
+    np.random.seed(seed)
```

Comment:
directly seeding `np.random` is dangerous since this adds a global seed to the full NumPy module and would affect all other Mitiq functions based on numpy.

The trick is to define a local random state as done, e.g., in [this function](https://github.com/unitaryfund/mitiq/blob/0c0b7a0fe73f582aa1974c4ab3c49b9da0b5753e/mitiq/observable/observable.py#L99).

Reply:
thanks for the suggestion @andreamari, I've updated the function accordingly.